.github/dependabot.yml

```diff
@@ -136,6 +136,19 @@ updates:
         patterns:
           - "*"
     target-branch: main
+  - commit-message:
+      prefix: chore
+    directories:
+      - /
+      - /compose
+    package-ecosystem: docker-compose
+    schedule:
+      interval: monthly
+    groups:
+      backend-docker-compose-dependencies:
+        patterns:
+          - "*"
```

Comment:
Will it discover `compose/docker-compose.enterprise.yml` and friends?...
I guess only one way to find out :man_shrugging: